jenkins/opensearch/maven-publish-1.3.x.jenkinsfile

```diff
@@ -20,11 +20,6 @@ pipeline {
     environment {
         AGENT_X64 = 'Jenkins-Agent-AL2023-X64-C54xlarge-Docker-Host'
     }
-    triggers {
-        parameterizedCron '''
-            H 1 * * * %INPUT_MANIFEST=1.3.15/opensearch-1.3.15.yml
```

Comment:
I believe we need to switch to new upcoming 1.x version. 1.3.19 in this case?

Reply:
Since there is such a long time without switching, I think it is ok for release manager to manually trigger if needed.

Else, I can change it to run 1.3.19 for now.

Thanks.

Reply:
Adding @zelinh into the conversation as he worked on 1.3.18.

Thanks.

Reply:
Yeah, as for me, I manually triggered this job when necessary. We should change it to the next release version as a post release activity. 

Could you maybe change this to 1.3.19 then as I think core should have been bumped to the next version.

Reply:
Sure, change to 1.3.19 version now.
Thanks.